hazelcast/src/main/java/com/hazelcast/map/impl/eviction/ExpirationManager.java

```diff
@@ -182,6 +190,10 @@ public synchronized void start() {
                 scheduleWithRepetition(task, taskPeriodSeconds, taskPeriodSeconds, SECONDS);
     }
 
+    /**
+     * Ends scheduling of the task that clears expired entries.
+     * Calling this method multiple times has same effect.
+     */
     public synchronized void stop() {
```

Comment:
Seems like `stop` may be private now, there are no external calls to it anymore. Not sure about `start`, the only external call is from `com.hazelcast.map.impl.MapManagedService#init`, which probably may be replaced with listening for life cycle events.

Reply:
done all.

Reply:
Great, thank you.